networkx/algorithms/tree/distance_measures.py

```diff
@@ -0,0 +1,129 @@
+import networkx as nx
+
+__all__ = [
+    "tree_centroid",
+]
+
+
+def _subtree_sizes(G, root):
+    """Return a `dict` of subtree sizes in a tree rooted at a given node.
+
```

Comment:
Can you add a paragraph (blank line after the first line then the paragraph) that describes exactly what "subtree" you are talking about and what you mean by "size".  The words "size" is used for many things -- and there are lots of subtrees in a tree.

Actually, this is a private function -- so maybe we shouldn't even have a doc_string.  But I guess it can help and doesn't hurt. So if it is easy to come up with a sentence or two saying what subtree and how "size" is defined, that's be nice.

Reply:
Number of nodes in the subtree, including the root of the subtree.